examples/README.md

```diff
@@ -0,0 +1,16 @@
+# Examples
+
+This folder contains a couple of example files to use for testing the MLFMU tools and some FMUs that have been generated using this tool.
+Each folder is set up to contain:
+
+* `config`: containing the `.onnx` and `interface.json` files
+* `generated_fmu`: binary FMU files, to serve as examples
+* `<FmuName>`: source files for the FMU
```

Comment:
Maybe mention that this source files is what in generated using the tool. 
Now a user might mistakenly think that this is something you have to write yourself?
But not a big issue.